### PR TITLE
fixed xr.supportsSession to xr.isSessionSupported for API change in example

### PR DESCRIPTION
**Pull request by himorin**
I suppose this line needs a fix...

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -162,7 +162,7 @@ NOTE: This means that {{XRSessionMode/"immersive-ar"}} sessions support all the
 The following code checks to see if {{XRSessionMode/"immersive-ar"}} sessions are supported.
 
 <pre highlight="js">
-navigator.xr.supportsSession('immersive-ar').then(() => {
+navigator.xr.isSessionSupported('immersive-ar').then(() => {
```
Comment (toji) [inline index.bs:165]: `isSessionSupported()` resolves to a boolean which needs to be checked.

```suggestion
navigator.xr.isSessionSupported('immersive-ar').then((supported) => {
  if (!supported) { return; }
```

**Review by toji — CHANGES_REQUESTED** [review]
Thanks for catching this! One minor tweak needed.

**Review by Manishearth — APPROVED** [review]
r+ with brandon's change

File: index.bs (hunk 1 above)
Comment (himorin) [reply index.bs]: Ah, yes. haven't read example other than function name...

**Pull request merged**